src/main/java/com/puppycrawl/tools/checkstyle/gui/CodeSelectorPresentation.java

```diff
@@ -45,22 +43,22 @@
      * Constructor.
      * @param ast ast node.
      * @param lines2position list to map lines.
+     * @noinspection AssignmentToCollectionOrArrayFieldFromParameter
      */
     public CodeSelectorPresentation(DetailAST ast, List<Integer> lines2position) {
```

Comment:
argument type should ImmutableList, in other case it is not clear why we suppress violation